tests/units/test_parse_pieces.py

```diff
@@ -49,14 +70,35 @@
 """
 
 PARSED_BUILD_ANSIBLE_VERSION = '2.10'
-PARSED_BUILD_ANSIBLE_BASE_VERSION = '2.10.1'
+PARSED_BUILD_ANSIBLE_CORE_VERSION = '2.10.1'
 PARSED_BUILD_DEPS = {
     'ansible.netcommon': '>=1.2.0,<2.0.0',
     'ansible.posix': '>=1.1.0,<2.0.0',
     'ansible.windows': '>=1.0.0,<2.0.0',
     'dellemc.os10': '>=1.0.0,<1.1.0'
 }
 
+BUILD_2 = """
+_ansible_version: 2.10
+# this is a comment
+_ansible_base_version: 2.10.1
```

Comment:
This looks purposeful, so I probably just don't understand how this works/fits together. But I'll note again the use of `_base` here.

Reply:
```suggestion
_ansible_core_version: 2.10.1
```